File: backend-dotnet/src/BiteRight.Application/Queries/Categories/Search/SearchResponse.cs
using Auth0.ManagementApi.Paging;
using BiteRight.Application.Dtos.Categories;
using BiteRight.Application.Dtos.Common;

namespace BiteRight.Application.Queries.Categories.Search;

public class SearchResponse
{
    public PaginatedList<CategoryDto> Categories { get; set; }
    
    public SearchResponse(
        PaginatedList<CategoryDto> categories
    )
    {
        Categories = categories;
    }
}

File: mobile/app/src/main/java/com/sobczal2/biteright/data/api/responses/categories/SearchResponse.kt
package com.sobczal2.biteright.data.api.responses.categories

import com.google.gson.annotations.SerializedName
import com.sobczal2.biteright.dto.categories.CategoryDto
import com.sobczal2.biteright.dto.common.PaginatedList

data class SearchResponse(
    @SerializedName("categories") val categories: PaginatedList<CategoryDto>
)
